lib/logger.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import logging.config
+
+class SeleniumFilter(logging.Filter):
+    def filter(self, record):
+        selenium_log = 'selenium.webdriver.remote.remote_connection'
+        return selenium_log not in record.getMessage()
```

Comment:
This section will remove much of the verbose debug-level logging that Selenium populates in the logs. It can be removed if you ever want to see things like requests made to the selenium server during bot activity. I felt like it was distracting since there's typically not many problems with Selenium in projects.